proposals/0201-high-level-interface-overlay-controllers.md

```diff
@@ -394,3 +439,26 @@ This proposal will add a total new high level interface layer abstracting many p
 ## Alternatives considered
 
 As discussed in the steering committee meeting from March 20 (see [here](https://github.com/smartdevicelink/sdl_evolution/issues/379#issuecomment-374736496)) this proposal is a counterproposal to [0133 - Enhanced iOS Proxy Interface](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0133-EnhancediOSProxyInterface.md).
+
+Regarding `SDLSliderController` there is a different behavior depending on the existance of a message string. In the main proposal it is proposed to show either value strings *or* the message. The alternative solution would be to always show value strings and concatenate the message into each value string if provided. The concatenation would be `@"%ld\n%@"` where `%ld` is the value and `%@` is the message. The upside is that we can merge translated values including a message. The downside is that the message is repeated for each value. Example:
```

Comment:
Please refrain from using we. Apologies I missed this in the first review.

`We` to `SDLC`